app/src/test/java/com/github/globant/githubsubscribers/subscriberslist/SubscribersListInteractorTest.java

```diff
@@ -0,0 +1,141 @@
+
+    private SubscribersListPresenter presenter;
+
+    @Before
+    public void setupSubscriberDetailPresenter() {
+        MockitoAnnotations.initMocks(this);
+
+        // Setup mocking for Debug class
+        mockStatic(Debug.class);
+
+        // Get a reference to the class under test
+        presenter = new SubscribersListPresenterImpl(view, interactor);
+
+    }
+
+    @Test
+    public void getSubcriberListFromInteractorAndLoadIntoView() {
+
+        // Given a initialized Subscriber object with fake data and a list is created
+        List<Subscriber> subscriberList = new ArrayList<>(Arrays.asList(new Subscriber(login, id, avataUrl)));
+
+        // When detail presenter is called to get a Subscribers list
+        presenter.getSubscribersList();
+
+        // Then Subscribers list is loaded from Interactor and the callback is captured
+        verify(interactor).getSubscribersDataList(OnFinishedListenerCallBack.capture());
+
+        OnFinishedListenerCallBack.getValue().onResponse(subscriberList);
+
+        view.showSubscribersList(subscriberList);
+    }
+
+    /**
+     * Unit test to try when the service is unavailable
+     */
+    @Test
+    public void getNotConnectionSubscribersListRequestFromInteractor() {
+
+        // Given a type error with fake error message
+        ErrorMessagesHelper.TypeError typeError = ErrorMessagesHelper.TypeError.NO_CONNECTION;
+        String errorMessage = Constants.EXCEPTION_ERROR + ": " + typeError.toString();
+
+        // When detail presenter is called to get a Subscriber list
+        presenter.getSubscribersList();
+
+        // Then Subscribers list is loaded from Interactor and the callback is captured
+        verify(interactor).getSubscribersDataList(OnFinishedListenerCallBack.capture());
+
+        OnFinishedListenerCallBack.getValue().onFailure(errorMessage, typeError);
+
+        int messageId = ErrorMessagesHelper.getMessage(typeError);
+        view.showSubscribersError(messageId);
+    }
+
+    /**
+     * Unit test to try when the service answers with a empty or null subscriber list
+     */
+    @Test
+    public void getEmptySubscribersListRequestFromInteractor() {
+
+        // Given a type error with fake error message
+        ErrorMessagesHelper.TypeError typeError = ErrorMessagesHelper.TypeError.BAD_ANSWER;
+        String errorMessage = Constants.EXCEPTION_ERROR + ": " + typeError.toString();
+
+        // When detail presenter is called to get a Subscriber list
+        presenter.getSubscribersList();
+
+        // Then Subscribers list is loaded from Interactor and the callback is captured
+
+        verify(interactor).getSubscribersDataList(OnFinishedListenerCallBack.capture());
+
+        OnFinishedListenerCallBack.getValue().onFailure(errorMessage, typeError);
+
+        int messageId = ErrorMessagesHelper.getMessage(typeError);
+        view.showSubscribersError(messageId);
```

Comment:
Yo need to call `verify(view).method(param)`. Please refer to https://codelabs.developers.google.com/codelabs/android-testing/index.html?index=..%2F..%2Findex#5